tests/command_line/test_combine_experiments.py

```diff
@@ -423,3 +423,30 @@ def test_failed_tolerance_error(dials_regression, monkeypatch):
         script.run_with_preparsed(params, options)
     assert "Beam" in str(exc.value)
     print("Got (expected) error message:", exc.value)
+
+
+@pytest.mark.xfail(
+    strict=True,
+    raises=AssertionError,
+    reason="https://github.com/dials/dials/issues/1093",
+)
+def test_combine_imagesets(dials_data, tmpdir):
+    data = dials_data("l_cysteine_dials_output")
+
+    for command in (
+        f"dials.import {data}/l-cyst_01_000*.cbf output.experiments=sweep1.expt",
+        f"dials.import {data}/l-cyst_02_000*.cbf output.experiments=sweep2.expt",
```

Comment:
I think this makes for a better OS-agnostic test.  Happy to be overruled though.
```suggestion
        f"dials.import {data / 'l-cyst_01_000*.cbf'} output.experiments=sweep1.expt",
        f"dials.import {data / 'l-cyst_02_000*.cbf'} output.experiments=sweep2.expt",
```